river/command.zig

```diff
@@ -70,6 +70,8 @@ const command_impls = std.ComptimeStringMap(
         .{ "send-layout-cmd",           @import("command/layout.zig").sendLayoutCmd },
         .{ "send-to-output",            @import("command/output.zig").sendToOutput },
         .{ "send-to-previous-tags",     @import("command/tags.zig").sendToPreviousTags },
+        // TODO: or hide-cursor never|now|delay msec?
+        .{ "set-cursor-state",          @import("command/config.zig").setCursorState },
```

Comment:
Which variant is better? `hide-cursor never|now|delay`, or `set-cursor-state shown|hidden|auto-hide`?